include/small/rlist.h

```diff
@@ -31,6 +31,7 @@
  * SUCH DAMAGE.
```

Comment:
Please, use `Fixes tarantool/tarantool#10143` in commit msg. The ticket doesn't belong to this repo and we cannot directly link it like that.

Reply:
Fixed